### ISSUE TITLE
fortios_vpnssl_settings auth_timeout cannot be set to 0

### ISSUE THREAD

**Issue by MilesTails** [fortinetdev/terraform-provider-fortios#120]
Unable to set auth_timeout to zero under fortios_vpnssl_settings. Appears to be bug, not done any further troubleshooting at this stage.

Reproduce with by setting auth_timeout = 0 within fortios_vpnssl_settings and run twice.

**Comment by frankshen01** [comment]
Hi @MilesTails , thank you for the issue. It has been fixed and released, please see the latest version(v1.6.15): https://registry.terraform.io/providers/fortinetdev/fortios/latest. 

```
# cat maintst.tf
provider "fortios" {
  hostname = "192.168.52.177"
  insecure = "true"
  token    = "rGqsgj9Qmh3dwfQdc8hd3t3G6xG3N5"
}

resource "fortios_vpnssl_settings" "trname" {
  login_attempt_limit = 0
  login_block_time    = 70
  login_timeout       = 30
  port                = 443
  servercert          = "self-sign"
  auth_timeout        = 0
}

# terraform apply
  ----

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # fortios_vpnssl_settings.trname will be created
  + resource "fortios_vpnssl_settings" "trname" {
      + algorithm                     = (known after apply)
      + auth_session_check_source_ip  = (known after apply)
      + auth_timeout                  = 0
      + auto_tunnel_static_route      = (known after apply)
      + banned_cipher                 = (known after apply)
      + check_referer                 = (known after apply)
      + default_portal                = (known after apply)
      + deflate_compression_level     = (known after apply)
      + deflate_min_data_size         = (known after apply)
      + dns_server1                   = (known after apply)
      + dns_server2                   = (known after apply)
      + dtls_hello_timeout            = (known after apply)
      + dtls_tunnel                   = (known after apply)
      + force_two_factor_auth         = (known after apply)
      + header_x_forwarded_for        = (known after apply)
      + http_compression              = (known after apply)
      + http_only_cookie              = (known after apply)
      + http_request_body_timeout     = (known after apply)
      + http_request_header_timeout   = (known after apply)
      + https_redirect                = (known after apply)
      + id                            = (known after apply)
      + idle_timeout                  = (known after apply)
      + ipv6_dns_server1              = (known after apply)
      + ipv6_dns_server2              = (known after apply)
      + ipv6_wins_server1             = (known after apply)
      + ipv6_wins_server2             = (known after apply)
      + login_attempt_limit           = 0
      + login_block_time              = 70
      + login_timeout                 = 30
      + port                          = 443
      + port_precedence               = (known after apply)
      + reqclientcert                 = (known after apply)
      + route_source_interface        = (known after apply)
      + servercert                    = "self-sign"
      + source_address6_negate        = (known after apply)
      + source_address_negate         = (known after apply)
      + ssl_client_renegotiation      = (known after apply)
      + ssl_insert_empty_fragment     = (known after apply)
      + ssl_max_proto_ver             = (known after apply)
      + ssl_min_proto_ver             = (known after apply)
      + tlsv1_0                       = (known after apply)
      + tlsv1_1                       = (known after apply)
      + tlsv1_2                       = (known after apply)
      + tlsv1_3                       = (known after apply)
      + tunnel_connect_without_reauth = (known after apply)
      + tunnel_user_session_timeout   = (known after apply)
      + unsafe_legacy_renegotiation   = (known after apply)
      + url_obscuration               = (known after apply)
      + wins_server1                  = (known after apply)
      + wins_server2                  = (known after apply)
      + x_content_type_options        = (known after apply)
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

fortios_vpnssl_settings.trname: Creating...
fortios_vpnssl_settings.trname: Creation complete after 0s [id=VpnSslSettings]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

# terraform plan
  ----
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.

fortios_vpnssl_settings.trname: Refreshing state... [id=VpnSslSettings]

------------------------------------------------------------------------

No changes. Infrastructure is up-to-date.

This means that Terraform did not detect any differences between your
configuration and real physical resources that exist. As a result, no
actions need to be performed.
'''

'''
mytestFortiGate (settings) # show
config vpn ssl settings
    set servercert "self-sign"
    set auth-timeout 0
    set login-attempt-limit 0
    set login-block-time 70
    set port 443
end

```

**Comment by MilesTails** [comment]
Amazing! Thank you.
